### ISSUE TITLE
ranges::view::all & concepts viewable_range are different with std::ranges

### ISSUE THREAD

**Issue by JYLeeLYJ** [ericniebler/range-v3#1725]
Hello , today I found that ranges::view::all cannot accept a prvalue range. 
For example , ```R = std::vector<int>``` is not a ranges::viewable_range.

But the same usage with std::ranges is ok  , which shows difference with range-v3 . 
According to [range.all](http://eel.is/c++draft/range.all#general-2) , ``` all(std::vector<int>()) ``` should be correct ( it is an owning_view in std::ranges ) .
It is actually "viewable" . 

[(same code on godbolt)](https://godbolt.org/z/WPceafWxd)

```
#include <range/v3/view.hpp>

#include <ranges>
#include <vector>

void foo(){
    auto v1 = ranges::view::all(std::vector<int>());    // error
    auto v2 = std::ranges::views::all(std::vector<int>()); // std is ok , which gets an owning_view

    static_assert(std::ranges::viewable_range<std::vector<int>>);   // true
    static_assert(ranges::viewable_range<std::vector<int>>);  // false
}
```

Does we need to ensure any consistency between range-v3 & std::ranges ?

**Comment by brevzin** [comment]
Yeah, range-v3 doesn't implement [P2415](https://wg21.link/p2415) yet.

**Comment by JYLeeLYJ** [comment]
Thanks for reply ! 
Is there any plan to implement P2415 in future versions ?  

> Yeah, range-v3 doesn't implement [P2415](https://wg21.link/p2415) yet.

**Comment by eroller** [comment]
consistency with C++20 would be nice and owning_view would be a nice addition
